Source/Atomic/Resource/Image.cpp

```diff
@@ -1211,6 +1216,91 @@ bool Image::SaveJPG(const String& fileName, int quality) const
         return false;
 }
 
+bool Image::SaveDDS(const String& fileName) const
+{

```

Comment:
Brilliant!, though guard for Windows/OSX/Linux :)  We should introduce a ATOMIC_PLATFORM_DESKTOP define which makes this simple... not for this PR